kafka-rest/src/main/java/io/confluent/kafkarest/controllers/SchemaManagerImpl.java

```diff
@@ -144,6 +144,13 @@ private RegisteredSchema getSchemaFromSchemaVersion(
           e);
     }
 
+    if (schema.getSchema() == null) {
```

Comment:
@rayokota is this the expected change based on latest schema-registry change?

Reply:
```suggestion
    if (schema == null) {
```

Reply:
@rayokota this wouldn't work, as the test returns a non-null schema object.
https://github.com/confluentinc/kafka-rest/blob/040e2a9ab85e3f0fe9d8c0545b03ee1474c97d4f/kafka-rest/src/test/java/io/confluent/kafkarest/controllers/SchemaManagerImplTest.java#L697

Reply:
ok, how about  `if (schema == null || schema.getSchema() == null)`